lib/cartopy/tests/crs/test_spilhaus.py

```diff
@@ -0,0 +1,52 @@
+# Copyright Crown and Cartopy Contributors
+#
+# This file is part of Cartopy and is released under the BSD 3-clause license.
+# See LICENSE in the root of the repository for full licensing details.
+"""
+Tests for the HEALPix projection.
+"""
+from packaging.version import parse as parse_version
+import pyproj
+import pytest
+
+import cartopy.crs as ccrs
+from .helpers import check_proj_params
+
+
+common_arg = {
+    'azi=40.17823482',
+    'datum=WGS84',
+    'ellps=WGS84',
+    'k_0=1.414213562373095',
+    'lat_0=-49.56371678',
+    'lon_0=66.94970198',
+    'no_defs',
+    'units=m',
+}
+@pytest.mark.skipif(
+    parse_version(pyproj.__version__) <= parse_version("3.7.0"),
```

Comment:
I think this is actually dependent on the PROJ version like you mentioned, so can we get that out of pyproj somehow instead? i.e. I think I can build 3.7.0 with PROJ 9.6 and this would still work.

Reply:
Yeah, this is now implemented. I haven't try older version of pyproj, I will remove the condition and try again.

Reply:
You are right. Even old version of pyproj would work.